custom/icds/constants.py

```diff
@@ -0,0 +1,4 @@
+GMP_XMLNS = "http://openrosa.org/formdesigner/b183124a25f2a0ceab266e4564d3526199ac4d75"
```

Comment:
It had one https://github.com/dimagi/commcare-hq/blob/master/custom/icds/__init__.py